.pre-commit-config.yaml

```diff
@@ -9,6 +9,21 @@ repos:
     hooks:
     -   id: isort
         name: isort
+-   repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v1.1.1
+    hooks:
+      - id: mypy
+        exclude: ^tests/
+        args: [--strict, --strict-optional, --python-version=3.10]
```

Comment:
Why not 3.11? :)

Reply:
strict_optional is true by default:
https://mypy.readthedocs.io/en/stable/config_file.html#confval-strict_optional